test/avocado/selenium-login.py

```diff
@@ -50,16 +64,18 @@ def setUp(self):
             self.driver = selenium.webdriver.Firefox()
             guest_machine = 'localhost'
         else:
-            selenium_hub = os.environ["HUB"] if os.environ.has_key("HUB") else "localhost"
-            browser = os.environ["BROWSER"] if os.environ.has_key("BROWSER") else "firefox"
+            selenium_hub = os.environ["HUB"] if os.environ.has_key(
+                "HUB") else "localhost"
+            browser = os.environ["BROWSER"] if os.environ.has_key(
+                "BROWSER") else "firefox"
             guest_machine = os.environ["GUEST"]
             self.driver = selenium.webdriver.Remote(
                 command_executor='http://%s:4444/wd/hub' % selenium_hub, desired_capabilities={'browserName': browser})
 
-        self.driver.set_window_size(1024, 768)
+        self.driver.set_window_size(1400, 1200)
         self.driver.set_page_load_timeout(90)
-        self.driver.implicitly_wait(10)
-        self.default_try = 10
+        self.driver.implicitly_wait(1)
+        self.default_try = 40
         self.default_sleep = 1
```

Comment:
I can remove that, but main purpose of that is on line 98:
returned = WebDriverWait(baseelement, self.default_sleep)...
And It is better to not have it harcoded directly in function.
other time.sleep() I'll remove